### DISCUSSION TITLE
Great work but need some help with Door Sensor

### DISCUSSION THREAD

**Discussion by mannish-kumar** [matt8707/ha-fusion#307] [Q&A]
Hi,
First of all Thank You for coming up with such a great idea.

I have a door Sonoff Zigbee door sensor which by default reports the status as "on" if door is opened and "off" if door is closed.
I have gone ahead and changed the Show as setting from "opening" to "Door" under Entities. Now HA reports the status as Open and Closed for this entity and same when I add this button in HA.

But when I add this button in Fusion Dashboard it still reports as On and Off. Is there a way I can change it to Opena and Closed in Fusion Dashboard?
![Entity Settings](https://github.com/matt8707/ha-fusion/assets/87553561/b870ac33-a6da-489c-9831-55ce9cee288a)
![Entity Status](https://github.com/matt8707/ha-fusion/assets/87553561/e3361010-5ece-4070-a33a-4d296794a32b)
![Fusion_Status](https://github.com/matt8707/ha-fusion/assets/87553561/782a4ff3-d07e-4225-91c7-e680bc4813a0)
![HA  Status](https://github.com/matt8707/ha-fusion/assets/87553561/61b66067-3f11-42e7-a564-67124575c72f)

**Comment by matt8707** [comment]
related https://github.com/matt8707/ha-fusion/issues/70
